From 439d6bbd2fcc70392c35cc4235a5e8aabf1404ea Mon Sep 17 00:00:00 2001
From: Robin Sommer <robin@icir.org>
Date: Sat, 16 Mar 2019 08:26:28 -0700
Subject: [PATCH] Don't build clangd.

On macOS, I'm getting:

    Undefined symbols for architecture x86_64:
      "clang::clangd::detail::log(clang::clangd::Logger::Level, llvm::formatv_object_base const&)", referenced from:
          void clang::clangd::detail::log<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >(clang::clangd::Logger::Level, char const*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&&) in Conversion.cpp.o
    ld: symbol(s) not found for architecture x86_64

(Linux works).
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 37e3e87e..55f36940 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,15 +13,15 @@ add_subdirectory(change-namespace)
 add_subdirectory(clang-doc)
 add_subdirectory(clang-query)
 add_subdirectory(clang-move)
-add_subdirectory(clangd)
+# add_subdirectory(clangd)  -- Doesn't compile on macOS.
 add_subdirectory(include-fixer)
 add_subdirectory(pp-trace)
 add_subdirectory(tool-template)
 
 # Add the common testsuite after all the tools.
 if(CLANG_INCLUDE_TESTS)
-add_subdirectory(test)
-add_subdirectory(unittests)
+# add_subdirectory(test)   -- Needs clangd
+# add_subdirectory(unittests)
 endif()
 
 option(CLANG_TOOLS_EXTRA_INCLUDE_DOCS "Generate build targets for the Clang Extra Tools docs."
-- 
2.21.0

